### PR TITLE
feat: all the new practice details

### PR DESCRIPTION
**Pull request by admbtlr**
Adds details for mobile, web and data practices.

### CODE REVIEW

**Comment by jonallured** [comment]
I _think_ this is missing platform practice, no? /cc @joeyAghion

File: README.md [hunk 1]
```diff
@@ -49,13 +49,24 @@ Engineers typically work on a product team and may also belong to an _Engineerin
   - Slack: [#product-platform](https://artsy.slack.com/messages/product-platform) 🔒
```
Comment (joeyAghion) [inline README.md:49]: This channel can be updated to `#practice-platform` and the Practice lead can be updated to @ansor4 (above).

**Review by joeyAghion — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (admbtlr) [reply README.md:49]: Aha, thanks!

**Comment by dblandin** [comment]
Screen Shot 2020-12-16 at 12 10 41 PM

If @ansor4 isn't a feature, I don't know what else is... 😅

**Review by dblandin — APPROVED** [review]
🎉

**Pull request merged**